server/token.go

```diff
@@ -515,9 +529,6 @@ func (s *IDPServer) issueTokens(w http.ResponseWriter, r *http.Request, ar *Auth
 		return
 	}
 
-	now := time.Now()
```

Comment:
just moved a few lines below and renamed `now` to `iat` - which is more descriptive of its use